src/app/hooks/useDetectOrdinalInSignPsbt.ts

```diff
@@ -11,28 +11,26 @@ const useDetectOrdinalInSignPsbt = () => {
     let userReceivesOrdinal = false;
 
     if (parsedPsbt) {
-      await Promise.all(
-        parsedPsbt.inputs.map(async (input) => {
-          try {
-            const data = await getUtxoOrdinalBundle(network.type, input.txid, input.index);
-
-            const bundle = mapRareSatsAPIResponseToRareSats(data);
-            bundle.items.forEach((item) => {
-              // we don't show unknown items for now
-              if (item.type === 'unknown') {
-                return;
-              }
-              bundleItems.push(item);
-            });
-          } catch (e) {
-            // we get back a 404 if the UTXO is not found, so it is likely this is a UTXO from an unpublished txn
-            if (!isAxiosError(e) || e.response?.status !== 404) {
-              // rethrow error if response was not 404
-              throw e;
+      parsedPsbt.inputs.map(async (input) => {
```

Comment:
I removed the Promise.all function here as it has been breaking the displaying of the batch PSBTs (when there are dozens of txs)
<img width="1239" alt="Monosnap DevTools - chrome-extension:::fpdmgfoenhfmedphgpdfkkdfcnggphhn:popup html 2023-11-15 15-35-09" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/c3727fe3-4c6d-4012-93c0-2a8dbeea1d98">